pkg/controller/directvolumemigration/rsync.go

```diff
@@ -80,8 +80,8 @@ data:
     [{{ $pvc.Name }}]
         comment = archive for {{ $pvc.Name }}
         path = /mnt/{{ $.Namespace }}/{{ $pvc.Name }}
-        uid = root
-        gid = root
+        use chroot = no
```

Comment:
I don't know enough about rsync to make a meaningful ACK here but I guess it worked?

Reply:
It did:

> fake super
> Setting lqfake super = yesrq for a module causes the daemon side to behave as if the --fake-user command-line option had been specified. This allows the full attributes of a file to be stored without having to have the daemon actually running as root.

> The daemon must run with root privileges if you wish to use chroot, to bind to a port numbered under 1024 (as is the default 873), or to set file ownership. Otherwise, it must just have permission to read and write the appropriate data, log, and lock files.

> use chroot
> If lquse chrootrq is true, the rsync daemon will chroot to the lqpathrq before starting the file transfer with the client.

from https://linux.die.net/man/5/rsyncd.conf

Reply:
@dymurray @alaypatel07 for awareness when you guys get back from PTO